packages/components/src/text-input/text-input.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * WordPress dependencies
+ */
+import { useMergeRefs } from '@wordpress/compose';
```

Comment:
The compiler cannot find module '@wordpress/compose' or its corresponding type declarations.

For the FormGroup + ControlLabel components migration the method was manually copied into the directory.

https://github.com/WordPress/gutenberg/pull/28568#issuecomment-770091755

Is that something we want to do?

Reply:
This will be fixed soon by #32676 when we'll be able to add a `references` entry in the `components/tsconfig.json` for `@wordpress/compose`. Then the TS compiler will be able to find this.